LongWarOfTheChosen/Src/LW_Overhaul/Classes/XComGameState_LWOutpost.uc

```diff
@@ -265,7 +265,7 @@ function StateObjectReference CreateRebel(XComGameState NewGameState, XComGameSt
 			{
 				FacelessChance *= 0.6;
 			}
-			if (Unit.HasSoldierAbility('MindMerge'))
+			if (Unit.HasSoldierAbility('MindMerge') || Unit.HasSoldierAbility('Soulfire') || Unit.HasSoldierAbility('Insanity'))
```

Comment:
If you're doing this, wouldn't be best to make it configurable at this point? I think there can be a batter implementation there.